yt/odin/checks/lib/query_tracker_engine_liveness/__init__.py

```diff
@@ -41,13 +40,15 @@ def log_query(logger, query_id, state):
 
 def track_query(yt_client, logger, query_id, stage):
     last_log_time = time.time()
+    last_state = ""
     while True:
         state = get_query_state(yt_client, query_id, stage)
 
         current_time = time.time()
-        if current_time - last_log_time >= LOG_FREQUENCY:
+        if state != last_state:
             log_query(logger, query_id, state)
             last_log_time = current_time
+            last_state = state
 
         if check_terminal_state(state):
             return state
```

Comment:
Is it true that the exact time is so important - that you need to log more often than once per second?

Reply:
It is no so important. Once per second is a good balance

Reply:
Changed poll frequency from 0.1 to 1.0 seconds